src/main/java/com/hierynomus/smbj/share/DiskShare.java

```diff
@@ -190,6 +211,38 @@ public FileInfo getFileInformation(SMB2FileId fileId) throws SMBApiException, Tr
     }
 
     /**
+     * Get Share Information for the current Disk Share
+     * @return the ShareInfo
+     * @throws SMBApiException
+     */
+    public ShareInfo getShareInformation() throws TransportException, SMBApiException {
+    	final String path = "";
```

Comment:
Why have this variable? It is only used in the call and never set. Please remove.